alembic/versions/509652e21254_store_build_timestamp.py

```diff
@@ -0,0 +1,42 @@
+"""store build timestamp
+
+Revision ID: 509652e21254
+Revises: 258490f6e667
+Create Date: 2020-03-04 07:29:52.618091
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "509652e21254"
+down_revision = "258490f6e667"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "copr_builds", sa.Column("build_finished_time", sa.Integer(), nullable=True)
+    )
+    op.add_column(
+        "copr_builds", sa.Column("build_start_time", sa.Integer(), nullable=True)
+    )
+    op.add_column(
+        "copr_builds", sa.Column("build_submitted_time", sa.Integer(), nullable=True)
+    )
+    op.drop_column("copr_builds", "logs")
```

Comment:
hm, please don't drop that field

Reply:
We have to manually remove this from the generated schema right? Because the _copr_builds_ table in models.py does not have a column called _logs_ but it is in the old schema, which is why autogenerate removed it.

Reply:
oh, got it; let's leave that then, it's true that I moved logs from copr build table to the srpm build table